R/iNZChangeDataWin.R

```diff
@@ -1889,18 +1883,18 @@ iNZJoinWin <- setRefClass(
 
             prevTbl <<- glayout(homogeneous = FALSE)
 
-            string1 <- glabel("Preview of the original dataset")
+            string1 <- glabel(tr("data_preview_original"))
             originview <- gpagedtable(
                 data.frame(head(GUI$getActiveData(lazy = TRUE), 10), stringsAsFactors = TRUE)
             )
-            string2 <- glabel("Select join methods")
+            string2 <- glabel(tr("data_join_method"))
             jointypes <- list(
-                "Inner Join" = "inner_join",
-                "Left Join" = "left_join",
-                "Right Join" = "right_join",
-                "Full Join" = "full_join",
-                "Semi Join" = "semi_join",
-                "Anti Join" = "anti_join"
+                tr("data_inner") = "inner_join",
```

Comment:
Sadly R isn't smart enough to do this. Would need to use something like 'defaultMenu' in MenuBarWidget instead (`names(x) <- sapply(names(x), tr)`).